tasks/unscramble-words.md

```diff
@@ -0,0 +1,41 @@
+# Summary
+> Groups had two minutes to unscramble as many words as possible on a list of 24 randomly scrambled words. Each item on the list had only one correct answer, and groups received one point for each correct answer.
+
+# References
+> Engel D, Woolley AW, Jing LX, Chabris CF, Malone TW (2014) Reading the Mind in the Eyes or Reading between the Lines? Theory of Mind Predicts Collective Intelligence Equally Well Online and Face-To-Face. PLOS ONE 9(12): e115212. https://doi.org/10.1371/journal.pone.0115212
+
```

Comment:
Could we add the link to supporting information: https://doi.org/10.1371/journal.pone.0115212.s002